Sources/StoryFlow/Interfaces/Helpers/Testing.swift

```diff
@@ -11,12 +11,27 @@ extension InputRequiring where Self: UIViewController {
 
 extension OutputProducing where Self: UIViewController {
 
-    public init(nibName: String? = nil, bundle: Bundle? = nil, produce: @escaping (OutputType) -> ()) {
+    public init(
+        nibName: String? = nil,
+        bundle: Bundle? = nil,
+        produce: @escaping (OutputType) -> OutputType?
```

Comment:
```suggestion
        produceStub: @escaping (OutputType) -> OutputType?
```
maybe worth to expose the name to emphasise different behaviour?

Reply:
also to make it easier for compiler to differentiate which init is used